lib/Mongo/MongoClient.php

```diff
@@ -74,6 +90,8 @@ public function __construct($server = 'default', array $options = ["connect" =>
 
         $this->server = $server;
         $this->client = new Client($server, $options, $driverOptions);
+        $info = $this->client->__debugInfo();
```

Comment:
I agree with you but let's say the lib does not provide much getters :smile:. Anyway would be way better if we can get something done upstream.